botfront/imports/ui/components/form_fields/import.style.less

```diff
@@ -0,0 +1,23 @@
+    .button-select-label {
+        display: block;
+        margin-bottom: 1em;
+    }
+    .ui.buttons.options-button-group {
+        display: block;
+        width: max-content;
+        border-radius: 6px;
+        .ui.button.option-button {
+            border-left-color:rgb(193, 193, 193);
+            border-right-color: rgb(193, 193, 193);
+            color: rgb(101, 101, 101) !important;
+                    border-top-color: rgb(193, 193, 193);
+        border-bottom-color: rgb(193, 193, 193);
+        border-left-color: rgb(193, 193, 193);
+        border-right-color: rgb(193, 193, 193);
+            &.active {
+                background-color: rgb(193, 193, 193) !important;
+                font-weight: bold;
+                color: rgb(45, 45, 45) !important;
```

Comment:
@kenrilea Those `!important `are here in order to override the styles defined by semantic ui, right ? otherwise I think there might be a better way to applies styles

Reply:
yes, the important is only there to override semantic ui importants